### PR TITLE
Move minVisibleScale prepending logic from the core to the kothic

### PR DESCRIPTION
**Pull request by pastk**
As per https://github.com/organicmaps/organicmaps/issues/4314#issuecomment-1442367661

### CODE REVIEW

**Comment by pastk** [comment]
Added a `Dump priorities into separate mapcss files` commit.

**Comment by pastk** [comment]
Now priorities are loaded from the new files (which are automatically re-formatted).
Priorities could be compressed / re-spaced on demand (manually).

File: src/libkomwm.py [hunk 1]
```diff
@@ -299,7 +569,7 @@ def rule_sort_key(dict_):
                             first = 1
                         if str(dict_.get('text')) == 'none':
                             first = 2
-                    return (first, int(dict_.get('z-index', 0)))
+                    return (first, dict_.get('object-id'))
```
Comment (vng) [inline src/libkomwm.py:572]: nit: Update comment above.

**Review by vng — APPROVED** [review]

File: src/libkomwm.py (hunk 1 above)
Comment (pastk) [reply src/libkomwm.py:572]: done

**Pull request merged**